doc_src/faq.rst

```diff
@@ -340,6 +341,14 @@ This also means that a few things are unsupportable:
 - Non-monospace fonts - there is *no way* for fish to figure out what width a specific character has as it has no influence on the terminal's font rendering.
 - Different widths for multiple ambiguous width characters - there is no way for fish to know which width you assign to each character.
 
+.. _faq-unicode:
```

Comment:
Wrong link. Maybe "faq-terminal-glitches", tho this isn't linked anywhere and could also be left out.